starter-code/src/functions-and-arrays.js

```diff
@@ -31,7 +80,25 @@ var wordsArr = [
   'chaos',
   'fuel',
   'palace'
-];
+]; 
+function averageWordLength(array){
+if(array.length===0){
+  return undefined
+}
+var totalWordLength = 0;
+for(var i=0; i<array.length;i++){
+  totalWordLength = totalWordLength+array[i].length
+
+}
+  
+  return totalWordLength / array.length;
+
+}
+
+
+
+
+
 
 // Unique Arrays
 var wordsUnique = [
```

Comment:
For the exercices you did, well done! The code is clean, straight to the point.
But the one you haven't had time to do, you'll find a correction here : https://github.com/ironhack-paris-webdev-august-2018/lab-javascript-functions-and-arrays/blob/master/solution-code/src/functions-and-arrays.js 

Keep up the good work!